R/target_encoder.R

```diff
@@ -0,0 +1,34 @@
+#' target_encoder
+#'
+#' This function encodes categorical variables with average target values for each category.
+#'
+#' @param X_train A tibble representing the training data set containing some categorical features/columns.
+#' @param X_test A tibble representing the test set, containing some set of categorical features/columns.
+#' @param y A numeric vector or character vector representing the target variable. 
+#'  If the objective is "binary", then the vector should only contain two unique values.
+#' @param cat_columns A character vector containing the names of the categorical columns in the tibble
+#'  that should be encoding.
```

Comment:
There is a typo here, I had this originally as well. Should say "A character vector containing the names of the categorical columns in the tibble that should be en**coded**."